### ISSUE TITLE
[Bug]: Error on Codeium Xcode extension (the operation couldn’t be completed)

### ISSUE THREAD

**Issue by peme969** [intitni/CopilotForXcode#615]
### Before Reporting

- [X] I have checked FAQ, and there is no solution to my issue
- [X] I have searched the existing issues, and there is no existing issue for my issue

### What happened?

~~A bug happened!~~
Obv, i downloaded everything - installed the language server, signed in to my existent codeium acc, etc…
<img width="1226" alt="image" src="https://github.com/user-attachments/assets/9b5d3407-8c67-46ce-b226-712a4e1b9651" />
<img width="1226" alt="image" src="https://github.com/user-attachments/assets/f8389c5a-4958-4939-88a5-fb4c3110f603" />
<img width="1226" alt="image" src="https://github.com/user-attachments/assets/1f06d2f9-891d-4942-9766-8a99bfc4aa42" />
<img width="1226" alt="image" src="https://github.com/user-attachments/assets/cc8ab0c3-ba0a-45bf-ab18-505f4f9929c1" />
PS: i just downloaded this extension for Xcode (my first extension on Xcode btw) so theres probably something else i should do. (though i went through every tutorial i could find on this)

### How to reproduce the bug.

Basically, I just added some code like the one provided in the OG tutorial - `def fib(n)`.

### macOS version

15.1.1

### Xcode version

16.2

### Copilot for Xcode version

i dont have copilot for Xcode but i didnt find the issues tab in the codeium for Xcode repo…

**Comment by intitni** [comment]
Please try [this from the FAQ](https://copilotforxcode.intii.com/faq/app-faq/#q-the-extension-says-it-cant-connect-to-the-extension-service-or-the-extension-service-version-loading-never-ends)

If you had to use the last solution and you are using the fork, please update the file name and identifiers accordingly.

**Comment by peme969** [comment]
Nevermind, I just figured it out by setting up a launch agent and granting that launch agent accessibility permissions.
(basically, for others reading this, just setup up the launch agent 👍🏼)
